clients/fides-js/src/lib/gpp/us-notices.ts

```diff
@@ -125,38 +123,37 @@ export const setGppOptOutsFromCookieAndExperience = ({
   cmpApi,
   cookie,
   experience,
-  forceGpp,
 }: {
   cmpApi: CmpApi;
   cookie: FidesCookie;
   experience: PrivacyExperience;
-  forceGpp: boolean;
 }) => {
-  const sectionsChanged = new Set<{ name: string; id: number }>();
-  const { consent } = cookie;
+  const sectionsChanged = new Set<GPPSection>();
+  const {
+    privacy_notices: notices = [],
+    region: experienceRegion,
+    gpp_settings: gppSettings,
+  } = experience;
+  const usApproach = gppSettings?.us_approach;
   const gppRegion = deriveGppFieldRegion({
-    experienceRegion: experience.region,
-    usApproach: experience.gpp_settings?.us_approach,
+    experienceRegion,
+    usApproach,
   });
   const gppSection = FIDES_REGION_TO_GPP_SECTION[gppRegion];
 
-  if (!gppSection) {
-    if (
-      (forceGpp && !experience?.gpp_settings) ||
-      experience?.gpp_settings?.us_approach === GPPUSApproach.STATE
-    ) {
-      cmpApi.setApplicableSections([-1]);
-      return [];
-    }
+  if (
+    !gppSection ||
+    (experienceRegion === "us" && usApproach === GPPUSApproach.STATE)
+  ) {
```

Comment:
same fix needed in both places. we should consider consolidating these 2 methods!

Reply:
same as above ^

Reply:
we should at last consolidate a check for experienceRegion being in the US